build/config/android/config.gni

```diff
@@ -16,8 +16,8 @@ if (is_android) {
 
   if (!defined(default_android_sdk_root)) {
     default_android_sdk_root = "//third_party/android_tools/sdk"
-    default_android_sdk_version = "30"
-    default_android_sdk_build_tools_version = "30.0.2"
+    default_android_sdk_version = "31"
+    default_android_sdk_build_tools_version = "31.0.0"
```

Comment:
Will this work with the agp in scenario app and background image tests?

Reply:
I haven't tried it. Is the process to merge the PR in buildroot first, and then find out later while rolling to the engine?

Reply:
I guess we'll find out. It should fail now if there's a problem because we're not allowing Gradle to download arbitrary sdk components